meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -1,183 +1,213 @@
 inherit image_types
 
-# Images inheriting this class MUST define:
+#
+# Create a raw image that can by written onto a storage device using dd/etcher.
+#
 # RESIN_IMAGE_BOOTLOADER        - bootloader
-# RESIN_BOOT_PARTITION_FILES    - list of items describing files to be deployed on boot partition
+# RESIN_BOOT_PARTITION_FILES    - list of items describing files to be deployed
+#                               - on boot partition
 #                               - items need to be in the 'src:dst' format
 #                               - src needs to be relative to DEPLOY_DIR_IMAGE
 #                               - dst needs to be an absolute path
-#                               - if dst is ommited ('src:' format used), absolute path of src will be used as dst
-#
-# Optional:
-# RESIN_SDIMG_ROOTFS_TYPE       - rootfs image to be used [default: ext4]
-# RESIN_BOOT_SPACE              - size of boot partition in KiB [default: 40960]
-# RESIN_SDIMG_COMPRESSION       - define this to compress the final SD image with gzip, xz or bzip2 [default: empty]
-
-#
-# Create an image that can by written onto a SD card using dd.
+#                               - if dst is ommited ('src:' format used),
+#                               - absolute path of src will be used as dst
```

Comment:
I see this is a new line break in order to shorten the lines; was it supposed to start with "-" also?